src/iris/api.py

```diff
@@ -4041,8 +4041,19 @@ def create_email_message(self, application, dest, subject, body):
                 'body': body,
                 'destination': dest
             }
-            message_id = session.execute(sql, data).lastrowid
-            session.commit()
+            retries = 0
+            while retries < 3:
+                try:
+                    message_id = session.execute(sql, data).lastrowid
+                    session.commit()
+                except Exception as e:
+                    retries += 1
+                    if retries == 3:
+                        logger.exception("failed to create email message")
+                    sleep(0.2)
```

Comment:
I think this is fine, as `session.commit()` will probably emit a nested exception and logging that should give us context as to what went wrong. 

My only feedback here is maybe we should log the sql statement it was trying to execute when the exception occurred?

Reply:
`session.commit` is in the try block. It will arrive as `e`. We should capture the information in that exception, not just log a messages saying "failed to create..."

Reply:
The sqlalchemy exception information that is logged along with that message will contain the query and values that were tried. The point is moot since I am removing this message retry because after reconsidering since retrying a message that confirms an incident was claimed via email does not seem pressing enough especially if it is underlying database issues that caused the failure, to begin with.